styles/gauge.json

```diff
@@ -481,84 +352,31 @@
 			"minzoom": 11,
 			"features": [{
 				"type": "LineString",
-				"coordinates": [[10, 50], [80, 50]],
+				"coordinates": [[10, 50], [105, 50]],
 				"properties": {
 					"railway": "miniature"
 				}
 			}],
 			"caption": "Miniature"
-		},
-		{
-			"minzoom": 11,
-			"features": [{
-				"type": "LineString",
-				"coordinates": [[10, 50], [80, 50]],
-				"properties": {
-					"railway": "miniature",
-					"gauge": "64"
-				}
-			}],
-			"caption": "64mm"
-		},
-		{
-			"minzoom": 11,
-			"features": [{
-				"type": "LineString",
-				"coordinates": [[10, 50], [80, 50]],
-				"properties": {
-					"railway": "miniature",
-					"gauge": "89"
-				}
-			}],
-			"caption": "89mm"
-		},
-		{
-			"minzoom": 11,
-			"features": [{
-				"type": "LineString",
-				"coordinates": [[10, 50], [80, 50]],
-				"properties": {
-					"railway": "miniature",
-					"gauge": "127"
-				}
-			}],
-			"caption": "127mm"
-		},
-		{
-			"minzoom": 11,
-			"features": [{
-				"type": "LineString",
-				"coordinates": [[10, 50], [80, 50]],
-				"properties": {
-					"railway": "miniature",
-					"gauge": "184"
-				}
-			}],
-			"caption": "184mm"
-		},
-		{
-			"minzoom": 11,
-			"features": [{
-				"type": "LineString",
-				"coordinates": [[10, 50], [80, 50]],
-				"properties": {
-					"railway": "miniature",
-					"gauge": "190"
-				}
-			}],
-			"caption": "190mm"
-		},
-		{
+		},{
```

Comment:
linebreak

Reply:
no bonus I guess... I just cant get the complete chain running and end up with uninterpreted html code.